components/org.wso2.carbon.identity.discovery/src/test/java/org/wso2/carbon/identity/discovery/OIDProviderConfigResponseTest.java

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.discovery;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+public class OIDProviderConfigResponseTest {
```

Comment:
Is this testing a bean class? if so, try to get this covered by other classes which uses this class in their business logic

Reply:
@pulasthi7 Isn't that a bad practice ? Covering a different method other than the method that is unit test is written to. In my opinion it is okay to write unit test for getters and setters as well. What happen if we wrote a logic inside a getter or setter later. WDYT ?